docs/use-nodemon-in-docker-chs-development-dev-mode.md

```diff
@@ -0,0 +1,35 @@
+# How to use `nodemon` when running app in docker-chs-development "dev mode"
+
+## What is this about?
+
+For node applications, when you run them in "dev mode" and do a change then this application will be updated by coping over the changed files and  restarting the docker container (see file `Tiltfile.dev` for details)
```

Comment:
coping = copying

changed files and  restarting = double space

Reply:
Updated

Reply:
Updated